packages/messaging/ios/RNFBMessaging/RNFBMessaging+AppDelegate.m

```diff
@@ -66,6 +68,16 @@ - (void)observe {
   });
 }
 
+// used to signal that a javascript handler for background messages is set
+- (void)signalBackgroundMessageHandlerSet {
+  RNFBMessagingAppDelegate *sharedInstance = [RNFBMessagingAppDelegate sharedInstance];
+  [sharedInstance.conditionBackgroundMessageHandlerSet lock];
+  NSLog(@"signalBackgroundMessageHandlerSet sharedInstance.backgroundMessageHandlerSet was %@", sharedInstance.backgroundMessageHandlerSet ? @"YES" : @"NO");
```

Comment:
I'm afraid our logging is all over the place in the module, partly my fault.
But I believe the correct style is to use this shared utility:

> app/ios/RNFBApp/RNFBSharedUtils.h:#define DLog(fmt, ...) NSLog((@"%s [Line %d] " fmt), __PRETTY_FUNCTION__, __LINE__, ##__VA_ARGS__);

Reply:
Awesome. Do you know, will this end up in the system console log? I needed for debugging in development to correlate the log lines with the other iOS system logs to match up when notifications arrived to the phone with when the app responded. If ELog/DLog only write to the react native log, not the device logs, these statements are far less interesting and I will just remove them.

Reply:
Yes - they are wrapping NSLog so should be going to the same place.
If they were RCTLog calls they'd go to the react-native console but these should be in Console.app or Xcode native device output

Reply:
I see it wraps NSLog, perfect.